### DISCUSSION TITLE
Example policies in values.yaml

### DISCUSSION THREAD

**Discussion by apopaa** [sse-secure-systems/connaisseur#285] [Q&A] [answered]
Am I correct in assuming that from the default values.yaml provided, only the 'pattern: "docker.io/library/*:*"' policy is an example that can be safely removed? And maybe 'pattern: "*:*"' policy can be modified to fit user? But that 'pattern: "k8s.gcr.io/*:*"' and 'pattern: "docker.io/securesystemsengineering/*:*"' are required policies, for K8s and Connaisseur respectively?

**Comment by xopham** [comment, marked as answer]
@apopaa there is different strategies to go by here.

**Remove Policies**: Most straight forward without changing anything, you can only drop the `docker.io/library/*:*` (you can also drop the associated `docker_official` key of the `dockerhub_basics` validator) and `*:*` pattern (might want to create a more specific one for your repo though). Removing the latter will mean that any non-matching image to the specific patterns is denied.

**[Namespaced validation](https://sse-secure-systems.github.io/connaisseur/v2.1.0/features/namespaced_validation/)**: You could use namespaced validation to exclude the `kube-system` and `connaisseur` namespaces in which case you could drop the `k8s.gcr.io/*:*` and `docker.io/securesystemsengineering/*:*` as well.

**Side note**: Some k8s solutions like Kind or microk8s come with some own pods required. You might want to check for those pods (e.g. in the `kube-system` namespace) and either validate or `allow` those vendor-specific images to make sure Connaisseur does not block those.
